### PR TITLE
Limelight Implementation

### PR DESCRIPTION
**Pull request by alicekuznetsov**
Probably good to push now, but there are a number of Fairbotics changes here that we'd need to later remove from main

### CODE REVIEW

**Comment by alicekuznetsov** [comment]
Closes #283 
Closes #284

**Comment by TaylerUva** [comment]
Builds are failing due to photonlib. That should be removed seeing as we don't need it anymore

**Comment by TaylerUva** [comment]
@ACat701 are you going to push up Fairbotics changes to this branch or should we merge this first (so we have a record of pre fair vs after) and create a new branch for the fair?

**Comment by alicekuznetsov** [comment]
@TaylerUva All relevant fairbotics changes are now in this pr- all of the stuff we don't need anymore (due to the field sizing) has been removed :>

File: src/main/java/frc/robot/LimelightHelpers.java
Comment (TaylerUva) [inline src/main/java/frc/robot/LimelightHelpers.java:1]: Let's dump what we need of this into SuperCORE

**Review by TaylerUva — APPROVED** [review]

**Pull request merged**